tests/components/datadog/test_init.py

```diff
@@ -73,26 +60,26 @@ def test_datadog_setup_defaults(self, mock_connection, mock_client):
             },
         )
 
-        assert mock_connection.call_count == 1
-        assert mock_connection.call_args == mock.call(
-            statsd_host="host", statsd_port=8125
-        )
-        assert self.hass.bus.listen.called
+        assert mock_init.call_count == 1
+        assert mock_init.call_args == mock.call(statsd_host="host", statsd_port=8125)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)